.github/workflows/pr-tests-syft.yml

```diff
@@ -69,7 +69,12 @@ jobs:
         run: |
           pip install --upgrade tox packaging wheel --default-timeout=60
           python -c "import platform; import os; os.system('pip install jaxlib==0.3.14 -f https://whls.blob.core.windows.net/unstable/index.html') if platform.system().lower() == 'windows' else ''"
-
+      - name: Install MacOS dependencies
```

Comment:
I think the action `crazy-max/ghaction-setup-docker` we use below installs colima etc automatically.